web/src/components/OutputTemplate/OutputTemplateCell/OutputTemplateCell.tsx

```diff
@@ -0,0 +1,48 @@
+import type {
+  FindOutputTemplateById,
+  FindOutputTemplateByIdVariables,
+} from 'types/graphql'
+
+import type {
+  CellSuccessProps,
+  CellFailureProps,
+  TypedDocumentNode,
+} from '@redwoodjs/web'
+
+import OutputTemplate from 'src/components/OutputTemplate/OutputTemplate'
+
+export const QUERY: TypedDocumentNode<
+  FindOutputTemplateById,
+  FindOutputTemplateByIdVariables
+> = gql`
+  query FindOutputTemplateById($id: Int!) {
+    outputTemplate: outputTemplate(id: $id) {
+      id
+      name
+      version
+      effectiveDate
+      rulesGeneratedAt
+      createdAt
+      updatedAt
+    }
+  }
+`
+
+export const Loading = () => <div>Loading...</div>
```

Comment:
Not urgent at all but might be nice to define these in a common place somewhere since we reuse them?